### PR TITLE
ZED + AR tracking

### PR DESCRIPTION
**Pull request by matthew-reynolds**
Initial ZED + fiducial mark tracking setup.
 - Installation instructions
 - Demo launch file (Basically lifted straight from the ZED examples lol)

Also added some packages

### CODE REVIEW

File: q_pup_vision/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+
+#############
+## Testing ##
+#############
+
+if(CATKIN_ENABLE_TESTING)
+    # Lol testing
+endif()
+
```
Comment (wmmc88) [inline q_pup_vision/CMakeLists.txt]:
```suggestion
```

**Review by wraftus — DISMISSED** [review]
image

**Comment by matthew-reynolds** [comment]
This man has achieved new levels of post assignment fried brain

File: q_pup_vision/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+#############
+## Install ##
+#############
+
+# Install headers
```
Comment (wmmc88) [inline q_pup_vision/CMakeLists.txt:30]: what about installing launch files?

**Review by wmmc88 — CHANGES_REQUESTED** [review]

File: q_pup_vision/CMakeLists.txt (hunk 1 above)
Comment (matthew-reynolds) [reply q_pup_vision/CMakeLists.txt]: Done in 4fe6bc3

File: q_pup_vision/CMakeLists.txt (hunk 2 above)
Comment (matthew-reynolds) [reply q_pup_vision/CMakeLists.txt:30]: Done in 4fe6bc3

**Review by wmmc88 — APPROVED** [review]

**Pull request merged**